pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -5912,24 +5981,30 @@ def __init__(self, *args, **kwargs):
         self.legend_label_func = lambda qbn, row: ''
         super().__init__(*args, **kwargs)
 
-    def process_data(self):
-        super().process_data()
+    def extract_data(self):
+        super().extract_data()
```

Comment:
Did you check that splitting the process_data code here into extract_data and process_data doesn't crash the DynamicPhaseAnalysis which inherits from this class? I see you only updated CPhaseLeakageAnalysis.